salt/state.py

```diff
@@ -2002,12 +2005,17 @@ def reconcile_procs(self, running):
                                'changes': {}}
                     try:
                         with salt.utils.fopen(ret_cache, 'rb') as fp_:
-                            ret = msgpack.loads(fp_.read())
+                            ret = msgpack.loads(fp_.read(), encoding='utf-8')
```

Comment:
Not in the sense that I have tested it in this instance. :] Are you running into trouble?